lib/l10n/app_en.arb

```diff
@@ -8,5 +8,8 @@
   "start_survey": "Start Survey",
   "submit_survey": "Submit Survey",
   "today": "Today",
-  "answer_text_area_hint": "Your thoughts"
+  "answer_text_area_hint": "Your thoughts",
+  "answer_dropdown_option_very_fullfilled": "Very fullfilled",
+  "answer_dropdown_option_somewhat_fullfilled": "Somewhat fullfilled",
+  "answer_dropdown_option_somewhat_unfullfilled": "Somewhat unfullfilled"
```

Comment:
Could you please recheck the survey "ibis Bangkok Riverside"? 

It's the second survey in the list, this survey does have a "dropdown" question.

So, should we add the answers from the API instead of hardcoding them? 🙏